lib/proximity.js

```diff
@@ -190,6 +190,36 @@ var Controllers = {
       }
     }
   },
+  HCSR04I2C: {
+    initialize: {
+        value: function(opts, datahandler) {
+			var BYTES_TO_READ = 0x02;
+            var address = opts.address || 0x27;
+            var msToNextRead = 90;
+
+            opts.address = address;
+
+            // set up IO connection:
+            this.io.i2cConfig(opts);
+
+			var read = function() {
+				setTimeout(function() {
+					this.board.io.i2cReadOnce(address, BYTES_TO_READ, function(data) {
+						datahandler((data[0] << 8) + data[1]);
+                        read();
+					});
+				}.bind(this), msToNextRead);
+			}.bind(this);
+
+            read();
+        }
+    },
```

Comment:
Nit: The indentation is inconsistent. Two spaces per tab is what most of the library has. 

@rwaldron I think jshint used to catch this. Can/should we add this to .jscsrc or is this a non-issue?